tests/components/plugwise/test_number.py

```diff
@@ -101,3 +105,30 @@ async def test_adam_temperature_offset_change(
     mock_smile_adam.set_number.assert_called_with(
         "6a3bf693d05e48e0b460c815a4fdd09d", "temperature_offset", 1.0
     )
+
+
+async def test_adam_temperature_offset_out_of_bounds_change(
+    hass: HomeAssistant, mock_smile_adam: MagicMock, init_integration: MockConfigEntry
+) -> None:
+    """Test changing of the temperature_offset number beyond limits."""
+    with pytest.raises(ServiceValidationError):
```

Comment:
```suggestion
    with pytest.raises(ServiceValidationError, match="..."):
```
Let's match on the error mesasge

Reply:
Added (4x)